Sources/Apodini/Properties/State.swift

```diff
@@ -56,12 +56,13 @@ extension State where Element: ExpressibleByNilLiteral {
 /// An `Activatable` element may allocate resources when `activate` is called. These
 /// resources may share information with any copies made from this element after `activate`
 /// was called.
-protocol Activatable {
+public protocol Activatable {
```

Comment:
Based on the changes in the last commits I would suggest that we can make `Activatable` and the `activate` function internal again:
```suggestion
protocol Activatable {
```